collab-service/Client/dist/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Not sure if your local repo is outdated, but what is the purpose of this file?

Reply:
Maybe I clicked save somewhere in VSCode. Can delete.